<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAddFilesDialog</class>
 <widget class="QDialog" name="CAddFilesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableView">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addBtn">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="qupod.qrc">
         <normaloff>:/icons/add</normaloff>:/icons/add</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delBtn">
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset resource="qupod.qrc">
         <normaloff>:/icons/delete</normaloff>:/icons/delete</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="icon">
        <iconset resource="qupod.qrc">
         <normaloff>:/icons/ok</normaloff>:/icons/ok</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="qupod.qrc">
         <normaloff>:/icons/cancel</normaloff>:/icons/cancel</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionAdd_file">
   <property name="icon">
    <iconset resource="qupod.qrc">
     <normaloff>:/icons/insert_file</normaloff>:/icons/insert_file</iconset>
   </property>
   <property name="text">
    <string>Add file</string>
   </property>
  </action>
  <action name="actionAdd_directory">
   <property name="icon">
    <iconset resource="qupod.qrc">
     <normaloff>:/icons/insert_directory</normaloff>:/icons/insert_directory</iconset>
   </property>
   <property name="text">
    <string>Add directory</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="qupod.qrc"/>
 </resources>
 <connections/>
</ui>
